tools/luthier.luau

```diff
@@ -251,6 +254,28 @@ local function generateStdLibFilesIfNeeded()
 	fs.close(hash)
 end
 
+local function generateTypeDefinitionFiles()
```

Comment:
This needs to do a hash check on the files and only run if needed, doesn't need the "If Needed" in the name, and honestly you can remove it from the generateStdLibFiles one too if you want.